.github/workflows/create_release.yml

```diff
@@ -0,0 +1,90 @@
+name: Create Release
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  create_release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - uses: actions/download-artifact@v3
+      with:
+        name: ni-grpc-device-client
+        path: downloads/ni-grpc-device-client
+
+    - uses: vimtor/action-zip@v1
```

Comment:
Download artifact unzips the artifact. So for windows we have to re-zip it. Is that right?

Reply:
Yes, or at least downloading the windows artifacts through these actions was just downloading the files/folders so we need to zip them up to include in the artifacts. The linux ones weren't zipped and instead are just single tar.gz files so they're fine to directly add to the artifacts. 

I'm not sure if they're zipped in the first place when the artifacts are uploaded or if it's just a directory. It might just be the directory and then when you click a link to one of them (say from a CI run) it zips them up and downloads that. Because from there, the linux ones are zipped too.

In any case, zipping them up in the workflow seems correct and produces artifacts in the draft release matching the form of the artifacts in our previous releases.